hack/verify-crds.sh

```diff
@@ -63,3 +63,20 @@ if [[ -n "${diff}" ]]; then
   exit 1
 fi
 echo "SUCCESS: generated CRDs up-to-date"
+
+# Verify that CRDs don't contain status fields as that causes issues when they
+# are managed by some CD tools. This check is necessary because currently
+# controller-gen adds a status field that needs to be removed manually.
+# See https://github.com/jetstack/cert-manager/pull/4379 for context
+crdPath="${tmpfiles}/deploy/crds"
+yq=$(realpath "$4")
+
+echo "Verifying that CRDs don't contain .status fields..."
+for file in ${crdPath}/*.yaml; do
+  name=$($yq e '.metadata.name' $file)
+  echo "Verifying that the CRD for $name does not contain status field.."
```

Comment:
nitpick(non-blocking): The two dots here look wrong to me; I'd prefer a full ellipsis (`...`) or else just not have anything at the end

```suggestion
  echo "Verifying that the CRD for $name does not contain a status field"
```

Reply:
Agree with the suggesion, it should hopefully never be so slow for the ellipsis to make sense